components/lwip/Kconfig

```diff
@@ -570,6 +570,31 @@ menu "LWIP"
 
     endmenu # UDP
 
+    menu "Checksums"
+
+    config LWIP_CHECKSUM_CHECK_IP
+        int "Enable LWIP IP checksums"
+        range 0 1
+        default 0
+        help
+            Enable checksum checking for received IP messages
+
+    config LWIP_CHECKSUM_CHECK_UDP
+        int "Enable LWIP UDP checksums"
+        range 0 1
+        default 0
+        help
+            Enable checksum checking for received UDP messages
+
+    config LWIP_CHECKSUM_CHECK_ICMP
+        int "Enable LWIP ICMP checksums"
+        range 0 1
+        default 0
```

Comment:
Current master has the `ICMP` checksums turned `ON` if i'm not mistaken, so suggest keeping the compatibility.